features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreFragment.kt

```diff
@@ -58,5 +60,34 @@ class ExploreFragment : Fragment(R.layout.fragment_explore) {
             viewModel.logEvent(AnalyticsConstants.CrowdNode.STAKING_ENTRY)
             viewModel.openStaking()
         }
+
+        setAPY(0.0) // hide the APY
+        viewModel.isBlockchainSynced.observe(viewLifecycleOwner) {
+            if (it == true) {
+                // update the APY when the blockchain is synced
+                viewModel.getStakingAPY()
+            }
+        }
+
+        viewModel.stakingAPY.observe(viewLifecycleOwner) {
+            setAPY(it)
+        }
+
+        // load the last APY value
+        viewModel.getLastStakingAPY()
+        viewModel.monitorBlockchainState()
```

Comment:
Goal:
1. Don't show the "Current APY = ?%" until there is a value to display
2. If there is a value that was saved from a previous calculation, then display it.
3. If the wallet is new (or reset) and the blockchain is syncing, then getLastStakingAPY returns an estimate
4. Once the blockchain is synced, we can calculate the Current APY